### ISSUE TITLE
Incorrect count when multiple dependency updates

### ISSUE THREAD

**Issue by sishbi** [raulpadilladelgado/check-dependencies-in-pr-action#2]
We have just started using your tool and find it very useful, but the calculation for the number of dependency updates does not match the list of dependencies.
For example:
```
You have 1 dependencies with newer available releases:
java-jwt 4.3.0 => 4.4.0
jackson-bom 2.15.0 => 2.15.2
jackson-dataformat-yaml 2.15.0 => 2.15.2
jackson-datatype-jsr310 2.15.0 => 2.15.2
```

I have submitted a fix in #1 to resolve this, please consider applying this PR.
Thanks!

**Comment by sishbi** [comment]
@raulpadilladelgado 
Hi there, please have a look at this and consider merging the PR.
Thanks!

**Comment by raulpadilladelgado** [comment]
Hi, I just merged and published it as version 1.0.1. Thank you for fixing it!
